libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/metering-point-type/dh-market-participant-actor-metering-point-type.component.html

```diff
@@ -0,0 +1,32 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container
+  *transloco="let t; read: 'marketParticipant.actor.create.masterData'"
+>
+  <mat-selection-list
+    #meteringPointTypeList
+    (selectionChange)="onModelChanged()"
+    [(ngModel)]="this.changes.meteringPointTypes"
```

Comment:
nitpick: remove `this`

```suggestion
    [(ngModel)]="changes.meteringPointTypes"
```